src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/Whitelist.java

```diff
@@ -73,11 +77,18 @@
     public static synchronized @Nonnull Whitelist all() {
         Jenkins j = Jenkins.getInstanceOrNull();
         if (j == null) {
+            LOGGER.warning("No Jenkins.instance");
             return new ProxyWhitelist();
         }
         Whitelist all = allByJenkins.get(j);
         if (all == null) {
-            all = new ProxyWhitelist(j.getExtensionList(Whitelist.class));
+            ExtensionList<Whitelist> allWhitelists = j.getExtensionList(Whitelist.class);
+            if (allWhitelists.isEmpty()) {
+                LOGGER.warning("Empty list of Whitelist");
```

Comment:
A stack trace might be interesting here:

```suggestion
                LOGGER.log(java.util.logging.Level.WARNING, null, new Exception("Empty list of Whitelist"));
```